### PR TITLE
Remove legacy examples

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

Remove legacy examples, sanity and e2e tests which based on examples
Replace custom mobilenet to torchvision in tests, removed ssd models
Bump accelerator to 1.9.0
removed modelzoo.md

### Tests
manual/job/post_training_quantization/688/
nightly/job/torch_nightly/609/
https://github.com/openvinotoolkit/nncf/actions/runs/16627298529
https://github.com/openvinotoolkit/nncf/actions/runs/16643653992

### CODE REVIEW

**Comment by ljaljushkin** [comment]
What about deleting these tests, since they rely on the deleted example folder?

- https://github.com/openvinotoolkit/nncf/blob/develop/tests/tensorflow/test_sanity_sample.py
- https://github.com/openvinotoolkit/nncf/blob/develop/tests/tensorflow/test_weekly.py
- https://github.com/openvinotoolkit/nncf/tree/develop/tests/tensorflow/data/configs

**Comment by AlexanderDokuchaev** [comment]
@ljaljushkin  removed

**Review by ljaljushkin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,8 +8,6 @@
 [Usage](#usage) •
 [Tutorials and Samples](#demos-tutorials-and-samples) •
 [Third-party integration](#third-party-repository-integration) •
-[Model Zoo](./docs/ModelZoo.md)
```
Comment (MaximProshin) [inline README.md]: I don't think we should simply remove it as this document also includes PTQ results and related references. Either keep the document but remove QAT only or replace it by new PTQ results document or smth else. cc @nikita-savelyevv who updated it last time and who could have some suggestions.

File: Makefile [hunk 2]
```diff
@@ -10,14 +10,6 @@ ifdef DATA
 	DATA_ARG := --data $(DATA)
 endif
 
-ifdef SOTA_DATA_DIR
```
Comment (MaximProshin) [inline Makefile:13]: Should Jenkins jobs/scripts be updated somehow? If so, will you create related tickets?

File: Makefile [hunk 3]
```diff
@@ -172,9 +164,6 @@ test-examples-torch:
 		--backend torch                     \
 		--junitxml ${JUNITXML_PATH}
 
-test-models-hub-torch:
```
Comment (MaximProshin) [inline Makefile:175]: Do these tests use old API only? Strangely, but I can't find these tests somehow. Can you share the link to these tests?

**Comment by daniil-lyakhov** [comment]
I just now realized that qat tests for the torch ptq pipeline were removed without any replacement https://github.com/openvinotoolkit/nncf/pull/3477. Should we perhaps introduce qat test for the torch ptq pipeline to test the claimed feature?

Looks like we do not need this examples after all, but we need some parts of it in torch2 tests. I suggest to open a ticket for that

File: README.md [hunk 4]
```diff
@@ -29,7 +27,7 @@ NNCF is designed to work with models from [PyTorch](https://pytorch.org/),
 [ONNX](https://onnx.ai/) and [OpenVINO&trade;](https://docs.openvino.ai).
 
 NNCF provides [samples](#demos-tutorials-and-samples) that demonstrate the usage of compression algorithms for different
-use cases and models. See compression results achievable with the NNCF-powered samples on the [NNCF Model Zoo page](./docs/ModelZoo.md).
```
Comment (MaximProshin) [inline README.md]: the same as above about PTQ.

File: README.md [hunk 5]
```diff
@@ -492,10 +477,6 @@ conda install -c conda-forge nncf
 System requirements of NNCF correspond to the used backend. System requirements for each backend and
 the matrix of corresponding versions can be found in [installation.md](./docs/Installation.md).
 
-## NNCF Compressed Model Zoo
```
Comment (MaximProshin) [inline README.md]: the same as above about PTQ results.

File: README.md [hunk 6]
```diff
@@ -425,19 +423,6 @@ Compact scripts demonstrating quantization and corresponding inference speed boo
 | [TensorFlow MobileNetV2](./examples/post_training_quantization/tensorflow/mobilenet_v2/README.md)                                        |            Post-Training Quantization            | TensorFlow |  Image Classification  |
 | [ONNX MobileNetV2](./examples/post_training_quantization/onnx/mobilenet_v2/README.md)                                                    |            Post-Training Quantization            |    ONNX    |  Image Classification  |
 
-### Training-Time Compression Examples
```
Comment (MaximProshin) [inline README.md:428]: I think Training-Time Compression section in the Usage above should also be removed. Or do you plan to update it with a new Sparsity API or what?

**Comment by daniil-lyakhov** [comment]
Please also remove links from the pruning openvino docs https://docs.openvino.ai/2025/openvino-workflow/model-optimization-guide/compressing-models-during-training/filter-pruning.html#examples

File: docs/PyPiPublishing.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ NNCF is designed to work with models from [PyTorch](https://pytorch.org/),
 [ONNX](https://onnx.ai/) and [OpenVINO&trade;](https://docs.openvino.ai).
 
 NNCF provides [samples](https://github.com/openvinotoolkit/nncf/blob/develop/#demos-tutorials-and-samples) that demonstrate the usage of compression algorithms for different
-use cases and models. See compression results achievable with the NNCF-powered samples on the [NNCF Model Zoo page](https://github.com/openvinotoolkit/nncf/blob/develop/docs/ModelZoo.md).
```
Comment (MaximProshin) [inline docs/PyPiPublishing.md]: the same as above about PTQ results

File: docs/PyPiPublishing.md [hunk 8]
```diff
@@ -32,7 +32,6 @@ For more information about NNCF, see:
 - [Key Features](#key-features)
 - [Installation](#installation-guide)
 - [Third-party integration](#third-party-repository-integration)
-- [NNCF Compressed Model Zoo](#nncf-compressed-model-zoo)
```
Comment (MaximProshin) [inline docs/PyPiPublishing.md:35]: the same as above about PTQ results

File: docs/PyPiPublishing.md [hunk 9]
```diff
@@ -114,10 +113,6 @@ NNCF may be easily integrated into training/evaluation pipelines of third-party
   NNCF is integrated into OpenVINO Training Extensions as a model optimization backend. You can train, optimize, and
   export new models based on available model templates as well as run the exported models with OpenVINO.
 
-## NNCF Compressed Model Zoo<a id="nncf-compressed-model-zoo"></a>
```
Comment (MaximProshin) [inline docs/PyPiPublishing.md]: the same as above about PTQ results

File: docs/usage/training_time_compression/other_algorithms/Usage.md [hunk 10]
```diff
@@ -377,7 +377,3 @@ model = training_loop.run(
     configure_optimizers_fn=configure_optimizers_fn,
     dump_checkpoint_fn=dump_checkpoint_fn)
 ```
-
```
Comment (MaximProshin) [inline docs/usage/training_time_compression/other_algorithms/Usage.md:380]: I wonder if we should replace it by new examples with new Sparsity API. What is a plan here?

File: tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json [hunk 11]
```diff
@@ -1,5 +1,5 @@
 {
-    "543/fq_weights_1": {
```
Comment (MaximProshin) [inline tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json:2]: Can you please clarify why it was updated?

File: tests/openvino/native/models.py [hunk 12]
```diff
@@ -36,8 +34,6 @@ def get_torch_model_info(model_name: str) -> tuple[Callable, tuple[int]]:
         "mobilenet-v3-small": (mobilenet_v3_small, (1, 3, 224, 224)),
         "resnet-18": (ResNet18, (1, 3, 224, 224)),
         "inception-v3": (inception_v3, (1, 3, 224, 224)),
-        "ssd-vgg-300": (ssd_vgg300, (1, 3, 300, 300)),
```
Comment (MaximProshin) [inline tests/openvino/native/models.py:39]: Why was it removed?

File: tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json (hunk 11 above)
Comment (MaximProshin) [reply tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json:2]: We shouldn't probably keep data for 2024.1 as it's outdated

File: Makefile (hunk 2 above)
Comment (AlexanderDokuchaev) [reply Makefile:13]: Yes, i will update it, and some jobs will be disabled (like e2e)

File: Makefile (hunk 3 above)
Comment (AlexanderDokuchaev) [reply Makefile:175]: No, but this tests was removed some time ago

File: docs/usage/training_time_compression/other_algorithms/Usage.md (hunk 10 above)
Comment (AlexanderDokuchaev) [reply docs/usage/training_time_compression/other_algorithms/Usage.md:380]: yes, it will be replaced. But now it still works

File: tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json (hunk 11 above)
Comment (AlexanderDokuchaev) [reply tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json:2]: Update mobilenetv2, now used model from torchvision

File: tests/openvino/native/models.py (hunk 12 above)
Comment (AlexanderDokuchaev) [reply tests/openvino/native/models.py:39]: Used model from example

File: tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json (hunk 11 above)
Comment (AlexanderDokuchaev) [reply tests/openvino/native/data/2024.1/reference_scales/mobilenet-v2_mixed.json:2]: Yes, the folders with versions are used to overwrite only the references that depend on the version and currently 2024.1 used as base. 
It can be collapsed, i will do it in other pr

File: README.md (hunk 1 above)
Comment (nikita-savelyevv) [reply README.md]: Model Zoo consists mostly of models trained with QAT/Pruning/Sparsity. I personally don't see any reason to keep a handful of the rest of models obtained by PTQ.

Comment (MaximProshin) [reply README.md]: @nikita-savelyevv , what is your suggestion then?

Comment (nikita-savelyevv) [reply README.md]: To remove Model Zoo page. Perhaps, we can provide a link to OpenVINO metrics page.

Comment (12qwas34) [reply README.md]: Some comments:
- I would also recommend to add the link to [OpenVINO Hugging Face page](https://huggingface.co/OpenVINO#models)
- Anyway, NNCF should demonstrate results which not included in mentioned links. Some results are included in the Usage.md or ReadMe.md of examples. It is almost impossible to find these results. I would suggest to add per-model BKMs to demonstrate achievable results.

File: README.md [hunk 13]
```diff
@@ -425,19 +423,6 @@ Compact scripts demonstrating quantization and corresponding inference speed boo
 | [TensorFlow MobileNetV2](./examples/post_training_quantization/tensorflow/mobilenet_v2/README.md)                                        |            Post-Training Quantization            | TensorFlow |  Image Classification  |
 | [ONNX MobileNetV2](./examples/post_training_quantization/onnx/mobilenet_v2/README.md)                                                    |            Post-Training Quantization            |    ONNX    |  Image Classification  |
```
Comment (12qwas34) [inline README.md:427]: Please add QAT examples

**Review by 12qwas34 — COMMENTED** [review]
The PR description does not contain details about the impact of these changes on the user:
- provide a table of which scenarios will be removed and what migration recommendations will be offered.
- If some scripts will be removed, please indicate the reason for this.

File: README.md (hunk 1 above)
Comment (MaximProshin) [reply README.md]: I'm still confused. If we don't remove Pruning and Sparsity methods and related samples, why do we remove this doc and numbers from it?

File: tests/openvino/native/models.py (hunk 12 above)
Comment (MaximProshin) [reply tests/openvino/native/models.py:39]: I didn't get it. Let's discuss offline.

Comment (MaximProshin) [reply tests/openvino/native/models.py:39]: discussed offline. agreed to remove.

File: README.md (hunk 1 above)
Comment (MaximProshin) [reply README.md]: Regarding ModelZoo.md, discussed offline: Agreed to keep the document and remove the tables with QAT results only at this stage. If we decide to add a table with models from examples and its results, we will add them to this document. If we decide to create another doc or add the table to Readme, we will remove the ModelZoo then. Also agreed to add links to https://docs.openvino.ai/2025/about-openvino/performance-benchmarks/model-accuracy-int8-fp32.html and https://huggingface.co/OpenVINO#models to Useful Links in Readme. 

@andrey-churkin , please also decide what we should do with ONNX PTQ results - honestly speaking I don't know how they were received and I think it's better to refer to models from examples instead and to ONNX results from the validation when they establish ONNX validation.

File: README.md (hunk 13 above)
Comment (MaximProshin) [reply README.md:427]: @12qwas34 , regarding "I would suggest to add per-model BKMs to demonstrate achievable results.", do you suggest to have a separate table with examples and results per model or do you suggest to add results to each example?
